src/components/datasets/explore/ConceptInstance/ConceptInstance.vue

```diff
@@ -459,22 +460,24 @@
               @confirm-remove-linked-property="openLinkedPropertyModal"
             />
 
-            <concept-instance-static-property
-              label="Pennsieve Id"
-              :value="instance.id"
-            />
+            <div class="static-prop-section">
+              <concept-instance-static-property
+                label="Pennsieve id"
+                :value="instance.id"
+              />
 
-            <concept-instance-static-property
-              label="Created By"
-              :user="instance.createdBy"
-              :date="instance.createdAt"
-            />
+              <concept-instance-static-property
+                label="created by"
+                :user="instance.createdBy"
+                :date="instance.createdAt"
+              />
```

Comment:
Is there a specific reason why the labels on these shouldn't be capitalized? Not that it matters much, just noticed it :)